### PR TITLE
[charts/redis-ha] don't kill container during sync

### LINKED ISSUE #80: [chart/redis-ha][BUG] Readiness probe should wait until replica is completed loading
**Describe the bug**
Readiness probe should not mark redis replicas as ready until the Loading Dataset is complete

**To Reproduce**
1. Setup redis ha
2. hook up to load with reasonable traffic (or for loop)
3. spin up new replica (scale from 3->4)

**Expected behavior**
Connections should always succeed

**Additional context**
Instead of successful connections, we get:
ERROR: LOADING Redis is loading the dataset in memory

### PR DESCRIPTION
**Pull request by Eraac**
#### What this PR does / why we need it:
Consider the container healthy if redis sync is in progress with the master

#### Which issue this PR fixes
  - fixes #80

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by DandyDeveloper** [comment]
@Eraac Sorry for the delay in reviewing this. 

I thought by default if we don't get a PONG back we will fail? Let me triple check this.

File: charts/redis-ha/templates/_configs.tpl [hunk 1]
```diff
@@ -501,7 +501,7 @@
       {{- end}}
         ping
     )
-    if [ "$response" != "PONG" ]; then
+    if [ "$response" != "PONG" ] && [ "${response:0:7}" != "LOADING" ] ; then
```
Comment (DandyDeveloper) [inline charts/redis-ha/templates/_configs.tpl:504]: I'm confused on how this is supposed to work. We're saying in your code; if it's not PONG AND not LOADING then exit with error code 1. 

Isn't the objective to basically say if we receive something OTHER than PONG, exit and fail? Which is what the original is doing?

**Review by DandyDeveloper — CHANGES_REQUESTED** [review]

File: charts/redis-ha/templates/_configs.tpl (hunk 1 above)
Comment (Eraac) [reply charts/redis-ha/templates/_configs.tpl:504]: During sync, the response is `LOADING: ...`, if we `exit 1` during sync the container will be restarted

| response        | liveness           |
| ------------- |:-------------:|
| PONG      | ok |
| LOADING      | ok      |
| .* | not ok      |

**Comment by DandyDeveloper** [comment]
@Eraac Sorry, I can't believe its been 24 days already... Getting this merged in now. Waiting for the tests!

**Comment by DandyDeveloper** [comment]
@Eraac Sorry again, can you pull in master? Has some fixes for the CI.

**Pull request merged**